### PR TITLE
replace Pytorch MLP by sklearn in BORE

### PR DESCRIPTION
**Pull request by geoalgo**
*Description of changes:* Replace Bore MLP implementation by sklearn instead of pytorch. This removes the pytorch dependency and makes the AutoML benchmark ~12x faster at no cost of accuracy.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by aaronkl — APPROVED** [review]
LGTM
